modules/ROOT/pages/partner-manager-architecture.adoc

```diff
@@ -44,21 +43,18 @@ image::pm-high-level-arch.png[Partner Manager high-level architecture]
 
 == Partner Manager for CloudHub
 
-When deploying B2B integration message flows to CloudHub 1.0, Anypoint MQ uses queues between processing stages to add additional reliability to B2B message processing.
+When deploying B2B integration message flows to CloudHub, Anypoint MQ uses queues between processing stages to add additional reliability to B2B message processing.
 
 As of the first deployment from Partner Manager, runtime applications deploy to the CloudHub region specified in the business group’s default settings in Runtime Manager by default. Anypoint MQ queues automatically generate during the deployment process in the same or the nearest available region. Any new runtime applications deploy to the same region as the first deployment.
 
-image::partner-manager-cloud-deployment.png[Deploying Partner Manager on CloudHub]
+image::cloudhub-deployment.png[Architecture that shows a CloudHub deployment]
 
-== Partner Manager for On-premises Mule Instances
+== Partner Manager for Runtime Fabric
 
-When deploying B2B integration message flows to your own hosted on-premises Mule instance, the instance uses HTTP calls between processing stages during B2B message processing.
+When deploying message flows to Runtime Fabric, Partner Manager uses ActiveMQ queues (set up and configured by the customer) between processing stages to add additional reliability to B2B message processing. ActiveMQ destinations (queues) are automatically created when the first transaction is processed through the runtime applications. 
```

Comment:
I try to avoid calling the user the "customer".

Any way to make passive active?